components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/APIEntity.java

```diff
@@ -115,6 +115,10 @@
     @CollectionTable(name = "ord_labels", joinColumns = @JoinColumn(name = "api_definition_id"))
     private List<Label> labels;
 
+    @ElementCollection
+    @CollectionTable(name = "api_definition_extensible", joinColumns = @JoinColumn(name = "api_definition_id"))
+    private List<ExtensibleEntity> extensibleEntity;
```

Comment:
For consistency reasons with the spec, this field must be renamed to `extensible`